ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74__ocp-4-13.yaml

```diff
@@ -10,12 +10,9 @@ build_root:
     tag: stackrox-test-0.3.57
 releases:
   latest:
-    prerelease:
-      architecture: amd64
-      product: ocp
-      version_bounds:
-        lower: 4.13.0-0
-        upper: 4.13.0-rc.5
+    release:
+      channel: fast
+      version: "4.13"
```

Comment:
Did you intend to switch to `fast` here?

Also please don't forget about
- release-4.0
- release-x.y
- release-3.73
- nightlies

Reply:
Also, it seems that `4.13.0` no-rc is out in https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.0/
How do you feel about going directly for https://issues.redhat.com/browse/ROX-16901 instead of bumping rc to 8 as ROX-17070 intended to?

Reply:
https://github.com/openshift/release/pull/39638 (linking for posterity)